modules/ticket_selector/TicketSelectorRowStandard.php

```diff
@@ -147,7 +147,7 @@ public function getHtml()
         $ticket_selector_row_html = EEH_HTML::tr(
             '',
             '',
-            "tckt-slctr-tbl-tr {$this->status_class}{$this->ticket_datetime_classes} "
+            "tckt-slctr-tbl-tr ee-hidden-ticket-tr {$this->status_class}{$this->ticket_datetime_classes} "
```

Comment:
would have been nice to apply BEM here as well
`tckt-slctr-tbl-tr ee-ticket-tr--hidden` or `tckt-slctr__tbl-tr tckt-slctr__tbl-tr--hidden`

Reply:
oh i agree... SASS > CSS imho